resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,6 +238,56 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class AlwaysShowScrollbars {
+    [DscProperty(Key)] [BinarySettingState] $ShowScrollBars = [BinarySettingState]::KeepCurrentValue
+
+    hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
+
+    [AlwaysShowScrollbars] Get() {
+        $currentState = [AlwaysShowScrollbars]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty)) {
+            $ShowBars = [BinarySettingState]::Disabled                
+        } else {
+			$ShowBars = (Get-ItemProperty -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty).DynamicScrollbars

```

Comment:
use camelCase for local variables and rename to `$dynamicScrollbarsValue`

Reply:
Is there documentation of this standard?